data/fields/seamark/type.json

```diff
@@ -4,5 +4,333 @@
+            },
+            "cable_area": {
+                "title": "Cable Area",
+                "description": "An area which contains one or more submarine cables."
+            },
+            "cable_overhead": {
+                "title": "Overhead Cable",
+                "description": "An assembly of wires or fibres, or a wire rope or chain, which is supported by structures such as poles or pylons and passing over or nearby navigable waters."
+            },
+            "cable_submarine": {
+                "title": "Submarine Cable",
+                "description": "An assembly of wires or fibres, or a wire rope or chain which has been laid underwater or buried beneath the seabed."
+            },
+            "causeway": {
+                "title": "Causeway",
+                "description": "A raised way across low or wet ground or water."
+            },
+            "checkpoint": {
+                "title": "Checkpoint",
+                "description": "An official place to register, declare or check goods and people."
+            },
+            "coastguard_station": {
+                "title": "Costguard Station",
+                "description": "Watch keeping station at which a watch is kept either continuously, or at certain times only."
+            },
+            "control_point": {
+                "title": "Control Point",
+                "description": "A point on the ground where position (horizontal and vertical) is used as a base for a dependent survey. Also referred to as a control station."
+            },
+            "crane": {
+                "title": "Crane",
+                "description": "A machine used for hoisting and moving heavy objects."
+            },
+            "distance_mark": {
+                "title": "Distance Mark",
+                "description": "A distance mark indicates the distance measured from an origin and consists of either a solid visible structure or a distinct location without special installation."
+            },
+            "dredged_area": {
+                "title": "Dredged Area",
+                "description": "An area of the bottom of a body of water which has been deepened by dredging."
+            },
+            "dumping_ground": {
+                "title": "Dumping Ground",
+                "description": "A sea area where dredged material or other potentially more harmful material, e.g. explosives, chemical waste, is deliberately deposited."
+            },
+            "fairway": {
+                "title": "Fairway",
+                "description": "That part of a river, harbour and so on, where the main navigable channel for vessels of larger size lies."
+            },
+            "ferry_route": {
+                "title": "Ferry Route",
+                "description": "A route in a body of water where a ferry crosses from one shoreline to another."
```

Comment:
we have https://wiki.openstreetmap.org/wiki/Tag:route%3Dferry - why we would want to promote seamark duplicate?

Yes, I know that some people making seamark tags created duplicates of established ones. It is not a good reason to accept or promote them.